dspace-xmlui/src/main/webapp/themes/dri2xhtml-alt/core/forms.xsl

```diff
@@ -770,6 +777,25 @@
         </xsl:choose>
     </xsl:template>
 
+    <!-- build the select field with the language options -->
+    <xsl:template name="languageSelectBuilder">
+        <xsl:param name="name"/>
+        <xsl:param name="selected"/>
+        <select>
+            <xsl:attribute name="name"><xsl:value-of select="concat($name, '[lang]')"/></xsl:attribute>
+               <xsl:for-each select="../dri:params/dri:param[@name='langs']/dri:option | ./dri:params/dri:param[@name='langs']/dri:option">
```

Comment:
Here applies the same comment like in Mirage2: I cannot find a place where this template is called, so that the `../dir:params/` part is used.